myapp/config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'users/index'
```

Comment:
Rails標準のルーティングで良いように思います
```suggestion
  resources :users, only: :index
```